op_crates/web/lib.deno_web.d.ts

```diff
@@ -108,6 +108,13 @@ declare class EventTarget {
      *
      * The event listener is appended to target's event listener list and is not
      * appended if it has the same type, callback, and capture. */
+  /** addEventListener() method attaches an event handler to the document.
```

Comment:
This should be a continuation of the previous JSDoc block, just adding the example, with a clean line break before it.